src/main/java/com/google/opensesame/servlets/ProjectPreviewsServlet.java

```diff
@@ -0,0 +1,41 @@
+package com.google.opensesame.servlets;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+import com.google.opensesame.projects.ProjectPreview;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/project-previews")
+public class ProjectPreviewsServlet extends HttpServlet {
+
+  private final Gson gson = new Gson();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<Entity> testProjectEntities = new ArrayList<Entity>();
+    Entity testProjectEntity = new Entity("Project");
+    testProjectEntity.setProperty("repositoryName", "tensorflow/tensorflow");
+    testProjectEntity.setProperty("numMentors", 5);
+    testProjectEntities.add(testProjectEntity);
+
+    testProjectEntity = new Entity("Project");
+    testProjectEntity.setProperty("repositoryName", "kubernetes/kubernetes");
+    testProjectEntity.setProperty("numMentors", 1);
+    testProjectEntities.add(testProjectEntity);
```

Comment:
nit: use a new variable name for a new object. That makes it clearer that you're adding 2 completely different objects to the list rather than just slightly modifying one object and adding it twice.